packages/swapper/src/swappers/RelaySwapper/utils/getTrade.ts

```diff
@@ -236,7 +236,16 @@ export async function getTrade<T extends 'quote' | 'rate'>({
 
   const { data: quote } = maybeQuote.unwrap()
 
-  const { slippageTolerance, rate, currencyOut, timeEstimate } = quote.details
+  const { slippageTolerance, currencyOut, timeEstimate } = quote.details
```

Comment:
And same here, use our input/output util instead of Relay's rate which doesn't rely on the same values we would here (out amount)